massgov/mayflower
# Patternlab/DP-10362  MF Org subnav dropdowns are not keyboard accessible

## Description
Adds keyboard accessibility for org subnav

## Related Issue / Ticket

- [DP-10362](https://jira.mass.gov/browse/DP-10362)


## Steps to Test

1.  Visit the the organization page in Mayflower
2. Tab through the subnav menu

- Enter / Spacebar will open the menu
   - open items should be traversable with tab and arrows 
- Tab will skip to the next menu item

## Screenshots
![orgnav](https://user-images.githubusercontent.com/18662734/45632392-d51fea00-ba62-11e8-9ddf-b3fdcaded0c4.gif)
